x-pack/metricbeat/module/enterprisesearch/health/health_integration_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build integration
+
+package health
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/v7/libbeat/tests/compose"
+	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+)
+
+func TestFetch(t *testing.T) {
+	service := compose.EnsureUpWithTimeout(t, 570, "enterprise_search")
```

Comment:
Why such a long timeout? We usually only use 120 seconds with very few exceptions. I'm fine is this is an exception and I trust there's a good reason for it but I just want to be sure that it's not a leftover

Reply:
Reduced those to 300 sec, but Enterprise Search in general could take quite some time to start since it needs to create a few dozen indexes, etc during the initial start.

Reply:
All right :slightly_smiling_face: